### ISSUE TITLE
LogitGetter: add copy_weights init argument

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#223]
If True, the classifier's weights are copied, otherwise it will use a reference to those weights. Should default to True.
